cypress/support/pages/articles.pageObject.js

```diff
@@ -0,0 +1,23 @@
+import PageObject from '../PageObject';
+
+class ArticlesPageObject extends PageObject {
+  url = '/#/articles/';
+
+  get articleTitle() {
+    return cy.get('h1');
+  }
+
+  clickOnEditBtn() {
+    cy.getByDataCy('editBtn')
```

Comment:
The `clickOnEditBtn()` method clicks on the first edit button found, which may not be the correct one in case of multiple articles. Consider passing an argument to the method to specify which article to edit.